tasks/task_with_retry.rb

```diff
@@ -0,0 +1,21 @@
+# :nodoc:
+class TaskWithRetry < Zenaton::Interfaces::Task
+  include Zenaton::Traits::Zenatonable
+
+  def handle
+    puts 'TaskWithRetry starts'
+    raise 'Error in TaskWithRetry'
```

Comment:
In the PHP and Node examples there seems to be a sleep for 3s before raising the exception. Should we add it here too?

Reply:
Good catch! Thanks